src/three-components/shader-chunk/encodings_pars_fragment.glsl.ts

```diff
@@ -1,3 +1,18 @@
+/* @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Technically this file is from three.js, so shouldn't it use their license? Same with normalmap below.